packages/react-tinacms-editor/src/components/WysiwygEditor/index.tsx

```diff
@@ -19,10 +19,9 @@ limitations under the License.
 import * as React from 'react'
 import { useEffect, useRef, useState } from 'react'
 import styled from 'styled-components'
-import { Plugin } from '@tinacms/core'
```

Comment:
Old plugin in props was not being used I am not sure what was intent for it.

Currently I am using it to pass custom plugins to editor.